src/app/task-list/task-list-item/task-list-item.component.ts

```diff
@@ -1,5 +1,6 @@
-import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
+import { Component, OnInit, Input, Output, EventEmitter, TemplateRef } from '@angular/core';
```

Comment:
Do we really need this TemplateRef  import ?

Reply:
You’re right. We needed TemplateRef for open Modal method. But now this method is also useless, So, I removed both of them.